### PR TITLE
Deploy rollbacks

### PR DESCRIPTION
**Pull request by anthdm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # FFAAS
+145068ff-e155-4c75-ab9f-9dbef07c7ef6
```
Comment (ValentinMontmirail) [inline README.md]: Those were the deployID during the Live. They should probably not stay in the README.

File: pkg/api/server.go [hunk 2]
```diff
@@ -53,12 +54,13 @@ func handleStatus(w http.ResponseWriter, r *http.Request) {
 
 // CreateAppParams holds all the necessary fields to create a new ffaas application.
 type CreateAppParams struct {
-	Name string `json:"name"`
+	Name        string            `json:"name"`
+	Environment map[string]string `json:"environment"`
 }
 
 func (p CreateAppParams) validate() error {
 	if len(p.Name) < 3 || len(p.Name) > 20 {
```
Comment (ValentinMontmirail) [inline pkg/api/server.go:62]: If the error message is changed to 'less than 40 characters' then so should be the conditional test. 
It should be `if len(p.Name) < 3 || len(p.Name) > 40`

File: pkg/api/server.go [hunk 3]
```diff
@@ -53,12 +54,13 @@ func handleStatus(w http.ResponseWriter, r *http.Request) {
...
-		return fmt.Errorf("name of the application should be longer than 3 and less than 20 characters")
+		return fmt.Errorf("name of the application should be longer than 3 and less than 40 characters")
```
Comment (ValentinMontmirail) [inline pkg/api/server.go]: If the error message is changed to 'less than 40 characters' then so should be the conditional test.
It should be if `len(p.Name) < 3 || len(p.Name) > 40`

Comment (anthdm) [reply pkg/api/server.go]: Good catch! Thanks.

**Pull request merged**